Pipeline/pipeline.py

```diff
@@ -367,20 +418,45 @@ def hypothesis_test(self, kind="Komolgorov-Smirnov", p_value=0.05):
             # p_df.to_html(r'tables\Welch test.html')
         return p_df
 
-    def validation(self, X, y):
+    def validation(self, x, y):
+        """
+
+        Parameters
+        ----------
+        x : TYPE
+            DESCRIPTION.
+        y : TYPE
+            DESCRIPTION.
+
+        Returns
+        -------
+        train : pd.DataFrame
+            DESCRIPTION.
+        test : pd.DataFrame
+            DESCRIPTION.
+

```

Comment:
Improve this part of the docs given information about the method and its parameters.

Reply:
Yes, I still have to improve this and the others docstrings. I've just opened the style to complement it later.